docs/content.zh/docs/deployment/memory/network_mem_tuning.md

```diff
@@ -87,6 +83,12 @@ Flink 1.14 新引入的缓冲消胀机制尝试通过自动调整缓冲数据量
 
 此外，如果您想减少缓冲数据量使其低于缓冲消胀当前允许的量，您可能需要手动的设置缓冲区的个数。
 
+#### High parallelism
+
+Currently, the buffer debloating mechanism can work correctly only by some parallelism values limit. When the parallelism exceeds this value, then increasing checkpoint time and decreasing throughput can be observed. The best way to fix this situation is to increase the number of buffers(`taskmanager.network.memory.buffers-per-channel`, `taskmanager.network.memory.floating-buffers-per-gate`)
```

Comment:
Maybe something like this?
```suggestion
Currently, the buffer debloating mechanism might not perform correctly with high parallelism (above ~200) using the default configuration. If you observe reduced throughput or higher than expected checkpointing times we suggest to increase the number of floating buffers (`taskmanager.network.memory.floating-buffers-per-gate`) from the default value to at least the number equal to the parallelism.
```

Reply:
Plus please brake long lines so that resolving conflicts in the future is easier.

Reply:
I have updated the commit